test/views/ViewApplicationViewSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import controllers.ViewApplicationControllerSpec.{applicationId, applicationRequest}
+import models.requests._
+import play.twirl.api.HtmlFormat
+import viewmodels.ApplicationViewModel
+import views.behaviours.ViewBehaviours
+import views.html.ViewApplicationView
+
+import java.time.{Clock, Instant, ZoneOffset}
+import scala.collection.immutable.List.from
+
+class ViewApplicationViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "viewApplication"
+
+  val ruling: Application =
```

Comment:
Please rename to application instead, thanks. Also, in case you are not aware, there is a trait [ApplicationGenerator](https://github.com/hmrc/advance-valuation-rulings-frontend/blob/main/test-utils/generators/ApplicationGenerator.scala#L25) which can be extended and  has [arbitraryApplication](https://github.com/hmrc/advance-valuation-rulings-frontend/blob/main/test-utils/generators/ApplicationGenerator.scala#L27) so this can simply be the below:

`private val application: Application = arbitraryApplication.arbitrary.sample.value`